lib/modules/manager/poetry/extract.ts

```diff
@@ -37,7 +38,11 @@ export async function extractPackageFile(
       if (dep.currentValue) {
         pythonVersion = dep.currentValue;
       }
-      return null;
+      return {
+        ...dep,
+        datasource: DockerDatasource.id,
+        commitMessageTopic: 'Python',
```

Comment:
Copied this from https://github.com/zeshuaro/renovate/blob/feat/poetry-extract-python/lib/modules/manager/pyenv/extract.ts#L7